TF_Service_and_Tooling_Unity/Assets/TouchFree/Tooling/Prefabs/Cursors/RingCursor.prefab

```diff
@@ -300,7 +300,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
```

Comment:
I think we want this to be on by default (and overridden in the TouchFreeApp Scene)